apps/website/screens/migration/Components16MigrationPage.tsx

```diff
@@ -0,0 +1,322 @@
+      </DxcParagraph>
+    ),
+  },
+  {
+    title: "Added components",
+    content: (
+      <>
+        <DxcParagraph>The following component is now available to be used in your applications:</DxcParagraph>
+        <DxcTable>
+          <thead>
+            <tr>
+              <th>Component</th>
+              <th>Features</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>Avatar</td>
+              <td>
+                Visual element used to identify users, teams, or entities across the interface. It helps create a
+                recognizable and consistent user experience by visually representing people or objects through images,
+                icons, or initials.
+              </td>
+            </tr>
+          </tbody>
+        </DxcTable>
+        <DxcParagraph>
+          For more information about its usage and features, refer to{" "}
+          <Link href="/components/avatar" passHref legacyBehavior>
+            <DxcLink>its documentation</DxcLink>
+          </Link>
+        </DxcParagraph>
+      </>
+    ),
+  },
+  {
+    title: "Modified components",
+    content: (
+      <DxcParagraph>
+        Several components were redesigned and now have a different API or behavior. See below for details.
+      </DxcParagraph>
+    ),
+    // TODO: ADD AN EXAMPLE CONVERTING THE OLD APIs TO THE NEW ONES OR REDIRECT TO THE COMPONENTS' PAGES?
+    subSections: [
+      {
+        title: "ApplicationLayout",
+        content: (
+          <DxcParagraph>
+            The <Code>ApplicationLayout</Code> component and its compound components have been redesigned. The new API
+            focuses on improved flexibility, better responsiveness, and easier customization. The <Code>header</Code>{" "}
+            prop is no longer required, however it is highly encouraged to have either a <Code>Sidenav</Code> or a{" "}
+            <Code>Header</Code> in your application. The following sections will explain the changes in more depth.
+          </DxcParagraph>
+        ),
+        subSections: [
+          {
+            title: "ApplicationLayout.Header",
+            content: (
+              <>
+                <DxcParagraph>
+                  The <Code>Header</Code> component has been redesigned. The new API provides a simpler and more
+                  flexible approach:
+                </DxcParagraph>
+                <DxcBulletedList>
+                  <DxcBulletedList.Item>
+                    <b>v15 (old)</b>: The header was configured using props like <Code>underlined</Code>,
+                    <Code>content</Code>, and <Code>responsiveContent</Code>. Dropdowns were rendered using{" "}
+                    <Code>DxcHeader.Dropdown</Code>.
+                  </DxcBulletedList.Item>
+                  <DxcBulletedList.Item>
+                    <b>v16</b>: The new header uses <Code>branding</Code> for logo/app title, <Code>navItems</Code> for
```

Comment:
branding does not longer exist in the Header component instead you have `logo` and `appTitle`